README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
+
+## Getting started
```

Comment:
Probably need something in here to make it 100% obvious that changes should be made in `source` and NOT `build`?